db/tasks.php

```diff
@@ -89,5 +89,14 @@
         'dayofweek' => '*',
         'month'     => '*'
     ),
+    [
+        'classname' => 'tool_objectfs\task\check_filedir_size',
+        'blocking'  => 0,
+        'minute'    => '*',
+        'hour'      => '2',
```

Comment:
This doesnt seem right, this is every minute from 2am to 3am? Did you intend just once at 2am each night?

Reply:
updated